components/kyma-environment-broker/internal/storage/storage_test.go

```diff
@@ -491,6 +491,24 @@ func TestSchemaInitializer(t *testing.T) {
 					},
 				},
 			}
+			latestPendingOperation := internal.ProvisioningOperation{
+				Operation: internal.Operation{
+					ID:    "latest-id-pending",
+					State: orchestration.Pending,
+					// used Round and set timezone to be able to compare timestamps
+					CreatedAt:              time.Now().Truncate(time.Millisecond).Add(2 * time.Minute),
+					UpdatedAt:              time.Now().Truncate(time.Millisecond).Add(3 * time.Minute),
+					InstanceID:             fixInstanceId,
+					OrchestrationID:        orchestrationID,
+					ProvisionerOperationID: "target-op-id",
+					Description:            "description",
+					Version:                1,
+					ProvisioningParameters: internal.ProvisioningParameters{},
+					InstanceDetails: internal.InstanceDetails{
+						Lms: internal.LMS{TenantID: "tenant-id"},
+					},
+				},
+			}
 
```

Comment:
please add also a pending operation in the test:
```
t.Run("Instances", func(t *testing.T) {
		t.Run("Should create and update instance", func(t *testing.T) {```

with different lms tenant Id just to be sure that such operation is not taken into account when the storage fetches instance details.